src/app/messagefilter.h

```diff
@@ -60,6 +63,7 @@ class MessageFilter : public QSortFilterProxyModel
 private:
     bool m_events;
     bool m_topicMessages;
+    IrcBuffer* m_buffer;
```

Comment:
can you initialize this to nullptr? the others are initialized in the constructor's initializer list because this was originally written before C++11 support was unconditionally enabled...

    IrcBuffer* m_buffer = nullptr;